src/app/modules/group/components/group-remove-button/group-remove-button.component.html

```diff
@@ -1,20 +1,20 @@
 <div class="container" *ngIf="state$ | async as state">
   <p-progressSpinner
       class="spinner"
-      *ngIf="state === 'loading' || (loading$ | async)"
+      *ngIf="state.isFetching || (deletionInProgress$ | async)"
       [style]="{width: '24px', height: '24px'}"
   >
   </p-progressSpinner>
-  <span class="has-children" *ngIf="state === 'hasChildren'" i18n>
+  <span class="has-children" *ngIf="state.data" i18n>
```

Comment:
I would prefer you to be more explicit here (your `If="state.data"` both excluded undefined and the boolean.
So `*ngIf="state.isReady && state.data"`

Reply:
Fixed